cmake/toolchain-gcc.cmake

```diff
@@ -27,6 +27,11 @@ endif()
 set(COMPILER_FLAGS "")
 set(LINKER_FLAGS "")
 
+# Don't ignore user-set LDFLAGS
+if(DEFINED ENV{LDFLAGS})
```

Comment:
Could you also replicate this in [toolchain-clang.cmake](https://github.com/scp-fs2open/fs2open.github.com/blob/master/cmake/toolchain-clang.cmake)?
Then this would cover both compilers.